### PR TITLE
Fix determinism build

### LINKED ISSUE #14595: windows-determinism is broken on dev15-rc
#14445 broke the windows determinism build for dev15-rc, starting with https://ci.dot.net/job/dotnet_roslyn/job/dev15-rc/job/windows_determinism/13/.

### PR DESCRIPTION
**Pull request by jaredpar**
Another instasce of MSBuild not loading our bootstrap compiler.  Bug tracking that is:

    https://github.com/Microsoft/msbuild/issues/1183

closes #14595

### CODE REVIEW

**Comment by 333fred** [comment]
We'll need to port this to master as well, as #14445 is present there as well.

**Comment by jaredpar** [comment]
test determinism please

**Comment by jaredpar** [comment]
@333fred it should just make it there via the next merge.

**Comment by jaredpar** [comment]
CC @Pilchie, @ManishJayaswal for ask mode approval.  Test only change.

File: netci.groovy [hunk 1]
```diff
@@ -152,7 +152,7 @@ set TMP=%TEMP%
     }
   }
 
-  def triggerPhraseOnly = true
+  def triggerPhraseOnly = false
```
Comment (Pilchie) [inline netci.groovy]: What does this do?

Comment (jaredpar) [inline netci.groovy]: Causes the determinism suite to run at every PR. This bug is evidence it should be doing that.

i'd planned on turnig this on long ago, just forgot about it.

**Comment by Pilchie** [comment]
Approved

**Comment by jaredpar** [comment]
The windows_release_unit64_prtest test has passed.  It's just hung for 45+ minutes archiving.

**Pull request merged**